.github/CONTRIBUTING.md

```diff
@@ -181,7 +181,7 @@ This is clearer and enhances readability of your code! Get used to doing it!
 ### Control statements
 (if, while, for, etc)
 
-* All control statements must not contain code on the same line as the statement (`if (blah) return`)
+* No control statement may contain code on the same line as the statement (`if (blah) return`)
```

Comment:
The two statements are equivalent, and the first one is more snappy. Why change it?

Reply:
"All must not" sounds a bit weird when you can just say "None may".

Reply:
I can get down with that